var/spack/repos/builtin/packages/libxsmm/package.py

```diff
@@ -88,14 +89,14 @@ def manual_install(self, prefix):
         install_tree('include', prefix.include)
         if '~header-only' in spec:
             install_tree('lib', prefix.lib)
-        doc_path = prefix.share + '/libxsmm/doc'
+        doc_path = join_path(prefix.share, 'libxsmm', 'doc')
         mkdirp(doc_path)
-        for doc_file in glob('documentation/*.md'):
+        for doc_file in glob(join_path('documentation', '*.md')):
             install(doc_file, doc_path)
-        for doc_file in glob('documentation/*.pdf'):
+        for doc_file in glob(join_path('documentation', '*.pdf')):
             install(doc_file, doc_path)
-        install('README.md', doc_path)
-        install('LICENSE', doc_path)
+        install('version.txt', doc_path)
+        install('LICENSE.md', doc_path)
```

Comment:
Won't that break installation of old version?

Reply:
That's true. but I do not know how to distinct the step for for "older" vs. "1.82 onwards".